### PR TITLE
[unit-tests] Add unit tests for ProgressBar component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `ProgressBar` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local & CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/18343/39/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/18349/9/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

File: packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    beforeEach(() => {
+      store = createStore(items)
+      wrapper = getShallowWrapper(store, true)
+    })
+    it('should change oc-icon name to "expand_less"', () => {
+      const ocIcon = wrapper.find(selectors.ocIcon)
+
+      expect(ocIcon.props().name).toEqual('expand_less')
+    })
+    it('should have "Hide Details"', () => {
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js:143]:
```suggestion
    it('should show the text "Hide Details"', () => {
```

File: packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    beforeEach(() => {
+      store = createStore(items)
+      wrapper = getShallowWrapper(store)
+    })
+    it('should have "expand_more" as oc-icon name', () => {
+      const ocIcon = wrapper.find(selectors.ocIcon)
+
+      expect(ocIcon.props().name).toEqual('expand_more')
+    })
+    it('should have "Show Details"', () => {
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js:110]:
```suggestion
    it('should show the text "Show Details"', () => {
```

File: packages/web-app-files/src/components/Upload/ProgressBar.vue [hunk 3]
```diff
@@ -23,6 +23,7 @@
       <div class="uk-width-expand uk-text-truncate">
         <translate
           v-if="count === 1"
+          id="upload-progress-single"
```
Comment (kulmann) [inline packages/web-app-files/src/components/Upload/ProgressBar.vue:26]: Please also add `files-` as a prefix to all the newly introduced ids. Kind of serves as a namespace of the app. Good that you already have a component namespace (`upload-progress`). 👍

File: packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      expect(ocProgressStub.props().value).toEqual(totalProgress)
+      expect(ocProgressText.text()).toBe(`${totalProgress} %`)
+    })
+    it('should show uploading files count but not file names', () => {
+      const progressSingle = wrapper.find(selectors.progressSingle)
+      const progressMulti = wrapper.find(selectors.progressMulti)
+
+      expect(progressSingle.exists()).toBeFalsy()
+      expect(progressMulti.exists()).toBeTruthy()
+      expect(progressMulti.props().translateN).toEqual(2)
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/Upload/ProgressBar.spec.js:94]:
```suggestion
      expect(progressMulti.props().translateN).toEqual(items.length)
```
💭 Just a suggestion

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18430/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10Locks</strong> https://drone.owncloud.com/owncloud/web/18430/8/1
The following scenarios passed on retry:
- webUIWebdavLockProtection/move.feature:123
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/18442/9/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Review by pascalwengerter — APPROVED** [review]
LGTM

**Pull request merged**